### PR TITLE
[Doc] Add doc for enhanced show mv

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 1]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
-| id            | The ID of the materialized view.                             |
-| name          | The name of the materialized view.                           |
-| database_name | The name of the database to which the materialized view resides. |
-| text          | The statement used to create the materialized view.          |
-| rows          | The rows of data in the materialized view.                   |
+| **Return**                 | **Description**                                              |
+| -------------------------- | ------------------------------------------------------------ |
+| id                         | The ID of the materialized view.                             |
+| name                       | The name of the materialized view.                           |
+| database_name              | The name of the database to which the materialized view resides. |
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:33]: to which 改为 in which

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 2]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
+| refresh_type               | The refresh type of the materialized view, including ROLLUP, MANNUL, ASYNC, INCREMENTAL. |
+| is_active                  | Whether the materialized view state is active.               |
+| last_refresh_start_time    | The start time of the last refresh of the materialized view. |
+| last_refresh_finished_time | The end time of the last refresh of the materialized view.   |
+| last_refresh_duration      | The last refresh time of the materialized view. Unit: second.|
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:38]: the last refresh time 会以为是个时间点，建议 The time taken by the last refresh. Unit: seconds.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 3]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
+| last_refresh_state         | The last refresh status of the materialized view, including PENDING, RUNNING, FAILED, SUCCESS. |
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:39]: 建议：The status of the last refresh, including xxx

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 4]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
+| inactive_code              | The error code for the last failed refresh of the materialized view (if the materialized view state is not active). |
+| inactive_reason            | The error message for the last failed refresh of the materialized view (if the materialized view state is not active). |
+| text                       | The statement used to create the materialized view.          |
+| rows                       | The rows of data in the materialized view.                   |
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:43]: The number of data rows in xxx.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 5]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
+| name                       | The name of the materialized view.                           |
+| database_name              | The name of the database to which the materialized view resides. |
+| refresh_type               | The refresh type of the materialized view, including ROLLUP, MANNUL, ASYNC, INCREMENTAL. |
+| is_active                  | Whether the materialized view state is active.               |
+| last_refresh_start_time    | The start time of the last refresh of the materialized view. |
+| last_refresh_finished_time | The end time of the last refresh of the materialized view.   |
+| last_refresh_duration      | The last refresh time of the materialized view. Unit: second.|
+| last_refresh_state         | The last refresh status of the materialized view, including PENDING, RUNNING, FAILED, SUCCESS. |
+| inactive_code              | The error code for the last failed refresh of the materialized view (if the materialized view state is not active). |
+| inactive_reason            | The error message for the last failed refresh of the materialized view (if the materialized view state is not active). |
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:41]: 我看参数写的是 reason，你看看具体用 reason 还是个 message。如果是reason, The reason why the last refresh failed.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 6]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
+DISTRIBUTED BY HASH(`c_custkey`) BUCKETS 10
 PROPERTIES (
-"replication_num" = "3",
-"storage_medium" = "HDD"
+"replication_num" = "1",
+"in_memory" = "false",
+"storage_format" = "DEFAULT"
+);
+
+-- Create MV: customer_mv
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:70]: 下面sql 关键字 用全大写 更一致

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 7]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
+properties (
+    "replication_num" = "1"
+)
+as select
+              c_custkey, c_phone, c_acctbal, count(1) as c_count, sum(c_acctbal) as c_sum
+   from
+              customer
+   group by c_custkey, c_phone, c_acctbal;
+
+-- Create MV: customer_mv
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:83]: 下面的关键字全大写

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 8]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
+              c_custkey, c_phone, c_acctbal, count(1) as c_count, sum(c_acctbal) as c_sum
+   from
+              customer
+   group by c_custkey, c_phone, c_acctbal; 
+
+refresh materialized view customer_mv;
 ```
 
-Example 2: Show specific materialized views by matching the name
+Example 1: Show a specific materialized views.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:99]: 单复数：materialized view

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 9]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
-| database_name | The name of the database to which the materialized view resides. |
-| text          | The statement used to create the materialized view.          |
-| rows          | The rows of data in the materialized view.                   |
+| **Return**                 | **Description**                                              |
+| -------------------------- | ------------------------------------------------------------ |
+| id                         | The ID of the materialized view.                             |
+| name                       | The name of the materialized view.                           |
+| database_name              | The name of the database to which the materialized view resides. |
+| refresh_type               | The refresh type of the materialized view, including ROLLUP, MANNUL, ASYNC, INCREMENTAL. |
+| is_active                  | Whether the materialized view state is active.               |
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:35]: 说一下取值为 true 和 false

File: docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md [hunk 10]
```diff
@@ -27,81 +26,134 @@ Parameters in brackets [] is optional.
...
+| last_refresh_start_time    | The start time of the last refresh of the materialized view. |
+| last_refresh_finished_time | The end time of the last refresh of the materialized view.   |
+| last_refresh_duration      | The last refresh time of the materialized view. Unit: second.|
+| last_refresh_state         | The last refresh status of the materialized view, including PENDING, RUNNING, FAILED, SUCCESS. |
+| inactive_code              | The error code for the last failed refresh of the materialized view (if the materialized view state is not active). |
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW MATERIALIZED VIEW.md:40]: 这里是否有不同的 code 取值，我看下面的示例是 0，是固定为0吗。就类似 http code   2，4, 5 分布代表不同意思

**Review by amber-create — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by EsoragotoSpirit** [comment]
run starrocks_admit_test

**Comment by EsoragotoSpirit** [comment]
@mergify backport branch-3.0

**Pull request merged**